pkg/build/frontend/config.go

```diff
@@ -16,15 +17,21 @@ func GetConfig(c *cli.Context, version string) (config.Config, config.Edition, e
 	}
 
 	mode := config.Edition(c.String("edition"))
+	buildID := c.String("build-id")
+	packageVersion, err := config.GetGrafanaVersion(buildID, GrafanaDir)
```

Comment:
That basically checks the tag or falls back to what is configured in the package, right?

Reply:
yep yep! wait I'll clean it up a bit

Reply:
let me know it the new commit's clearer @zerok